specs/lib/loadObjSpec.js

```diff
@@ -341,7 +343,54 @@ describe('loadObj', function() {
             }), done).toResolve();
     });
 
-    it('does not process file with invalid contents', function(done) {
+    function getFirstPosition(data) {
+        var positions = data.nodes[0].meshes[0].positions;
+        return new Cartesian3(positions.get(0), positions.get(1), positions.get(2));
+    }
+
+    function getFirstNormal(data) {
+        var normals = data.nodes[0].meshes[0].normals;
+        return new Cartesian3(normals.get(0), normals.get(1), normals.get(2));
+    }
+
+    function checkAxisConversion(inputUpAxis, outputUpAxis, position, normal) {
+        var sameAxis = (inputUpAxis === outputUpAxis);
+        var options = clone(defaultOptions);
+        options.inputUpAxis = inputUpAxis;
+        options.outputUpAxis = outputUpAxis;
+        return loadObj(objRotatedUrl, options)
+            .then(function(data) {
+                var rotatedPosition = getFirstPosition(data);
+                var rotatedNormal = getFirstNormal(data);
+                if (sameAxis) {
+                    expect(rotatedPosition).toEqual(position);
+                    expect(rotatedNormal).toEqual(normal);
+                } else {
+                    expect(rotatedPosition).not.toEqual(position);
+                    expect(rotatedNormal).not.toEqual(normal);
```

Comment:
I guess it's not a big deal since we're using Cesium's up-axis conversion, which I assume is relatively well tested, but should we try to come up with more definitive tests for this? It could be a huge pain though.